build.gradle

```diff
@@ -1,11 +1,16 @@
 // Gradle plugins
 buildscript {
     repositories {
+        mavenLocal()
```

Comment:
If it is then it's broken because that will never work on the Jenkins nor Travis ;)